zebra-rpc/src/methods/tests/vectors.rs

```diff
@@ -612,3 +612,53 @@ async fn rpc_getaddressutxos_response() {
 
     mempool.expect_no_requests().await;
 }
+
+#[tokio::test(flavor = "multi_thread")]
+#[cfg(feature = "getblocktemplate-rpcs")]
+async fn rpc_getblockcount() {
```

Comment:
We can move this and future tests to a test modulo inside `getblocktemplate` if we want

Reply:
Yes please, we want to keep the getblocktemplate code and tests in separate modules.